opcodes/riscv-opc.c

```diff
@@ -2016,14 +2016,14 @@ const struct riscv_opcode riscv_opcodes[] =
 {"cv.dotup.b",            0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_DOTUP_B, MASK_CV_DOTUP_B, match_opcode, 0},
 {"cv.dotup.sc.h",         0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_DOTUP_SC_H, MASK_CV_DOTUP_SC_H, match_opcode, 0},
 {"cv.dotup.sc.b",         0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_DOTUP_SC_B, MASK_CV_DOTUP_SC_B, match_opcode, 0},
-{"cv.dotup.sci.h",        0, INSN_CLASS_COREV_SIMD, "d,s,b5", MATCH_CV_DOTUP_SCI_H, MASK_CV_DOTUP_SCI_H, match_opcode, 0},
-{"cv.dotup.sci.b",        0, INSN_CLASS_COREV_SIMD, "d,s,b5", MATCH_CV_DOTUP_SCI_B, MASK_CV_DOTUP_SCI_B, match_opcode, 0},
+{"cv.dotup.sci.h",        0, INSN_CLASS_COREV_SIMD, "d,s,b8", MATCH_CV_DOTUP_SCI_H, MASK_CV_DOTUP_SCI_H, match_opcode, 0},
+{"cv.dotup.sci.b",        0, INSN_CLASS_COREV_SIMD, "d,s,b8", MATCH_CV_DOTUP_SCI_B, MASK_CV_DOTUP_SCI_B, match_opcode, 0},
 {"cv.dotusp.h",           0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_DOTUSP_H, MASK_CV_DOTUSP_H, match_opcode, 0},
 {"cv.dotusp.b",           0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_DOTUSP_B, MASK_CV_DOTUSP_B, match_opcode, 0},
 {"cv.dotusp.sc.h",        0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_DOTUSP_SC_H, MASK_CV_DOTUSP_SC_H, match_opcode, 0},
 {"cv.dotusp.sc.b",        0, INSN_CLASS_COREV_SIMD, "d,s,t", MATCH_CV_DOTUSP_SC_B, MASK_CV_DOTUSP_SC_B, match_opcode, 0},
-{"cv.dotusp.sci.h",       0, INSN_CLASS_COREV_SIMD, "d,s,b5", MATCH_CV_DOTUSP_SCI_H, MASK_CV_DOTUSP_SCI_H, match_opcode, 0},
-{"cv.dotusp.sci.b",       0, INSN_CLASS_COREV_SIMD, "d,s,b5", MATCH_CV_DOTUSP_SCI_B, MASK_CV_DOTUSP_SCI_B, match_opcode, 0},
+{"cv.dotusp.sci.h",       0, INSN_CLASS_COREV_SIMD, "d,s,b8", MATCH_CV_DOTUSP_SCI_H, MASK_CV_DOTUSP_SCI_H, match_opcode, 0},
+{"cv.dotusp.sci.b",       0, INSN_CLASS_COREV_SIMD, "d,s,b8", MATCH_CV_DOTUSP_SCI_B, MASK_CV_DOTUSP_SCI_B, match_opcode, 0},
```

Comment:
Is this correct? The spec for `cv.dotusp[.sc,.sci].h` and `cv.dotusp[.sc,.sci].b` says the second source operand is signed.  It is the first source operation, which is signed, but this is always a 32-bit register anyway.

Reply:
Thanks - the change looks good.